CP1404/prac_06/guitar_test.py

```diff
@@ -0,0 +1,12 @@
+from guitar import Guitar
+
+
+def main():
+    test_guitar = Guitar("Gibson L-5 CES", 1922, 16035)
+
+    print(f"{test_guitar.name} get_age() - Expected 99 got {test_guitar.get_age()}")
+    print(f"{test_guitar.name} is_vintage() - Expected True got {test_guitar.is_vintage()}")
```

Comment:
These tests are incomplete. With only 1 guitar you don't know if they always return 99/True.